src/env/test/mod.rs

```diff
@@ -4,19 +4,39 @@ use crate::api::firmware_protection::FirmwareProtection;
 use crate::ctap::status_code::Ctap2StatusCode;
 use crate::ctap::Channel;
 use crate::env::{Env, UserPresence};
-use crypto::rng256::ThreadRng256;
+use crypto::rng256::Rng256;
 use persistent_store::{BufferOptions, BufferStorage, Store};
+use rand::rngs::StdRng;
+use rand::{Rng, SeedableRng};
 
 mod upgrade_storage;
 
 pub struct TestEnv {
-    rng: ThreadRng256,
+    rng: TestRng256,
     user_presence: TestUserPresence,
     store: Store<BufferStorage>,
     upgrade_storage: Option<BufferUpgradeStorage>,
     customization: CustomizationImpl,
 }
 
+pub struct TestRng256 {
```

Comment:
I was wondering before if RNG should be part of `crypto`, and this is another reason for separating it? So we don't have too many different places where we define some sort of RNG. Doesn't have to be right now, but I would like to hear your opinion.

(The other use case was a library that `crypto` was supposed to use, and that library needing RNG. Meaning a circular dependency because of RNG not being separated.)

Reply:
I think this case doesn't serve as an additional reason to separate it because it's normal that fuzzer/test define its own implementation of some public interface together in a test folder. But I see no disadvantage of separating them too.

Reply:
Yes, I agree that RNG should not be part of `crypto`.